stylesheets/commons/BigMatch.less

```diff
@@ -795,6 +796,73 @@ ul.match-bm-lol-game-veto-overview-pick {
 	}
 }
 
+.match-bm-game-veto-overview-team-veto {
+	padding: 1rem 0;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	gap: 0.5rem;
+
```

Comment:
The header is missing a border-bottom. So you can either add it here as border-top or create a new header class with `-lol`.

```suggestion
	border-top: 0.0625rem solid #bbbbbb;

	.theme--dark & {
		border-color: var( --clr-secondary-25 );
	}

```

Reply:
Oh and the text on light mode needs to be white 
![image](https://github.com/user-attachments/assets/ffd53c3a-ca22-4f7d-b92c-6e5f5e938504)